Directory.Build.props

```diff
@@ -124,52 +124,60 @@
 
   <!-- version number information -->
   <PropertyGroup>
-    <IosTargetFrameworkVersion>13.6</IosTargetFrameworkVersion>
-    <TvosTargetFrameworkVersion>13.6</TvosTargetFrameworkVersion>
-    <MacCatalystTargetFrameworkVersion>13.5</MacCatalystTargetFrameworkVersion>
-    <MacosTargetFrameworkVersion>10.14</MacosTargetFrameworkVersion>
-    <AndroidTargetFrameworkVersion>30.0</AndroidTargetFrameworkVersion>
+    <IosTargetFrameworkVersion>16.1</IosTargetFrameworkVersion>
+    <TvosTargetFrameworkVersion>16.1</TvosTargetFrameworkVersion>
+    <MacCatalystTargetFrameworkVersion>16.1</MacCatalystTargetFrameworkVersion>
+    <MacosTargetFrameworkVersion>13.0</MacosTargetFrameworkVersion>
+    <AndroidTargetFrameworkVersion>33.0</AndroidTargetFrameworkVersion>
```

Comment:
34 is the default in .NET 8. If you take this to the .NET 9 branch, you'll want to change it to 35.

Reply:
Yeah, that will have to be done.